acceleration/transform_speed.ipynb

```diff
@@ -38,11 +38,45 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "MONAI version: 0.9.1+61.g1befe23a\n",
```

Comment:
I think maybe better to remove these print log?
We removed in many other tutorial notebooks.

Thanks.